kuma/api/v1/urls.py

```diff
@@ -9,4 +9,9 @@
     re_path(r"^search/(?P<locale>[^/]+)/?$", views.search, name="api.v1.search"),
     re_path(r"^bc-signal/?$", views.bc_signal, name="api.v1.bc_signal"),
     re_path(r"^users/(?P<username>[^/]+)/?$", views.get_user, name="api.v1.get_user"),
+    re_path(
+        r"^subscriptions/?$",
```

Comment:
Oh I see. That's what a bunch of the other endpoints do. Sigh. Right vs. consistency... Hm...

Reply:
Yeah tbh https://twitter.com/peterbe/status/1244342578931212288

I didn't spare too many thoughts here, but happy to start the good fight of changing things for the better (the API endpoints do give us that opportunity).
Thanks for noticing!

Reply:
This might be a parent page though, given that subscriptions are a resource group and we might want to add resource routes like

`DELETE /subscriptions/2`

Reply:
The day we need `/subscriptions/2` we can change *this* line to be `r"^subscriptions/$",`

Reply:
So, can we drop the `?` and stick to `/`?

Reply:
Actually, I think you should put the `/` there. That way it becomes more future-proof for when we do things like `DELETE /api/v1/subscriptions/2`
If you accidentally use something like `fetch('/api/v1/subscriptions')` it will still work thanks to Django. 
So the `/` being there means we don't need to change this OR, line 9 in `subscription-form.jsx` that future day when we add more stuff to the API.